internal/pkg/build/sources/conveyorPacker_oci.go

```diff
@@ -144,11 +144,13 @@ func (cp *OCIConveyorPacker) Get(ctx context.Context, b *sytypes.Bundle) (err er
 	cp.sysCtx = &types.SystemContext{
 		OCIInsecureSkipTLSVerify: cp.b.Opts.NoHTTPS,
 		DockerAuthConfig:         cp.b.Opts.DockerAuthConfig,
+		DockerDaemonHost:         cp.b.Opts.DockerDaemonHost,
```

Comment:
@dtrudg is it not enough to add it here?

Reply:
At least in early testing, when I changed this cp.sysCtx variable to something nonsensical it was enough for it to try using it (and tell me localhost:pancakes wasn't reachable)!

Reply:
You need to set it into the build options struct in `build_linux.go` here, so that it is propagated from the flag in the CLI level, down to the build level. Note how `DockerAuthConfig` is set here:

https://github.com/sylabs/singularity/blob/702930d9c36c8ff80eabdf1481c5c03a462b1a70/cmd/internal/cli/build_linux.go#L383